transformato/state.py

```diff
@@ -29,6 +29,8 @@ def __init__(
             list of mutations defined by the transformato.ProposeMutationRoute object
         configuration : dict
             configuration dictionary
+        consecutive_runs : bool or int
+            defines how many runs should be performed per intst state, if not used it is False else give an integer number
```

Comment:
I'd prefer to have static types for variables if possible. in that specific case I'd suggest that we use `int` and `if =! 0` multiple runs are performed